<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>427</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="MplWidget" name="MplWidget" native="true">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>0</y>
           <width>501</width>
           <height>371</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
        <widget class="QCheckBox" name="company1_checkbox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Company 1</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="soldiers1_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>53</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Soldiers</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="apcs1_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>53</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>APCs</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="enemies_checkBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>53</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Enemies</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="enemies_soldiers_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>300</y>
           <width>53</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Soldiers</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="launchers_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>320</y>
           <width>53</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Launchers</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="lookout_points_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>340</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Lookout Points</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="soldiers2_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>53</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Soldiers</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="company2_checkbox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Company 2</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="apcs2_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>53</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>APCs</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="soldiers3_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>230</y>
           <width>53</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Soldiers</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="company3_checkbox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Company 3</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="apcs3_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>250</y>
           <width>53</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>APCs</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="commander1_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Commander</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="commander2_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Commander</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="commander3_checkbox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Commander</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
